src/api/docs/paths/webhook.ts

```diff
@@ -0,0 +1,63 @@
+import { serverError } from "../defaults";
+
+const WebhookFields = {
+    pedidoId: {
+        type: "string",
+    },
+    tipo: {
+        type: "string",
+        enum: ["aprovado", "recusado"],
+        default: "aprovado",
+    },
+};
+
+const RequiredFields = ["pedidoId", "tipo"];
+
+export const WebhookPaths = {
+    "/webhook/pagamento/mock": {
+        post: {
+            tags: ["webhook"],
+            summary:
+                "Rota que recebe um webhook de atualização do status de pagamento",
```

Comment:
Sugestão bem opcional, será se não vale a pena deixar descrito os tipos que aceitamos? Pra facilitar os professores testando.

Reply:
O "tipos que aceitamos" o enum no caso?

Se for o enum, tem na documentação em schema:
![image](https://github.com/karineriquena/fiap-soat1-tech-challenge/assets/46209283/a252b7f9-7559-4e3c-b254-31c84b873bce)

Reply:
atualizei o texto:

![image](https://github.com/karineriquena/fiap-soat1-tech-challenge/assets/46209283/806f1bfc-6345-4de6-82d6-d378c8d332fb)